appdatabase/migrations/sql/1676536757_add_fallback_rpc_to_networks.up.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE networks ADD COLUMN fallback_url VARCHAR NOT NULL DEFAULT "";
```

Comment:
In other migrations I saw that after adding a column, somethign like this is executed immediatly:
```
UPDATE networks SET fallback_url = "";
```

Reply:
If possible, please remove migrationsprevnodecfg/bindata.go from this commit as it wasnt really affected by the PR

Reply:
Yes to both, done